app/javascript/components/applications/application_details/ApplicationDetails.js

```diff
@@ -68,7 +68,7 @@ const ApplicationDetails = ({ application, fields, fileBaseUrl }) => {
       <ApplicationCard
         title='Declared Household Income'
         fields={declareHousholdIncome} />
-      { application.flagged_applications && (
+      { !isEmpty(application.flagged_applications) && (
```

Comment:
I tried to check that application on full but looks like it was deleted? But to test I did pull this application up:`/applications/a0o0U000000XDKzQAO`

It doesn't have flagged applications and the table doesn't show up as expected.

Reply:
ok interesting. good to go then!

Reply:
@akegan On the application page, did we want to prevent entire sections from showing up when there is no content available for them, or are we choosing to still show the section header when there is no content for the section?

Reply:
@jrubenoff do you have an opinion here on whether we should hide the section entirely when no data is available for it, or if we should still show the section header? it seems like as an eng team we're leaning towards hiding the section entirely, but we want to validate this with design

Reply:
Ideally we should show a blank state, but hiding the section is fine for now

Reply:
We definitely agree that that would be better. I thought we had a ticket for improving our display of blank fields in the application view page somewhere but I'm having trouble finding it. @jrubenoff do you remember if you already made one? If not, I can create a new one

Reply:
I don't think there's a story for it